quizara-backend/models/leaderboard.model.js

```diff
@@ -0,0 +1,16 @@
+const mongoose  = require("mongoose")
+const  {Schema} = mongoose
+
+const LeaderboardSchema = new mongoose.Schema({
+      name:  String,
```

Comment:
While defining Schema always wrap the property in `Object` i.e `name:{
type: String,
required: true
}}`

so that you can add multiple properties later.